localization_launch/launch/util/util.launch.xml

```diff
@@ -2,13 +2,13 @@
 <launch>
 
   <!-- Topics -->
-  <arg name="input_sensor_points_topic" default="/sensing/lidar/top/rectified/pointcloud" description="input topic name for raw pointcloud"/>
+  <arg name="input_sensor_points_topic" description="input topic name for raw pointcloud"/>
```

Comment:
input/pointcloud

Reply:
discussed with @taichiH

Reply:
I changed arg names both launch and universe.
- https://github.com/tier4/autoware_launch/pull/215/commits/97588d74a6b3948c6415e64b20951a65d554e231
- https://github.com/autowarefoundation/autoware.universe/pull/433/commits/22757e6c4a382ef7d1b5cfd4adbe8922fa3d8164